module/input/Camera/src/Camera.cpp

```diff
@@ -231,13 +234,27 @@ namespace input {
             arv_stream_set_emit_signals(stream.get(), true);
         });
 
+        on<Trigger<Sensors>>().then("Buffer Sensors", [this](const Sensors& sensors) {
```

Comment:
Why not use a `Last<N>`?

Reply:
Because I wouldn't have a way of passing that to the static function